frontend/src/components/roadmap/sections/RoadmapHeader.tsx

```diff
@@ -0,0 +1,49 @@
+import { useTranslations } from "next-intl";
+import { Grid, GridContainer } from "@trussworks/react-uswds";
+
+export default function RoadmapHeader() {
+  const t = useTranslations("Roadmap");
+
+  return (
+    <div className="text-white bg-primary-darkest padding-top-4 tablet:padding-y-6">
+      <GridContainer>
+        <Grid row>
+          <Grid
+            tablet={{ col: 6 }}
+            className="display-flex flex-column flex-justify-center"
+          >
+            <h1 className="font-sans-xl margin-0 tablet:font-sans-2xl tablet:maxw-2">
+              {t("pageHeaderTitle")}
+            </h1>
+            <p
+              className={
+                "font-sans-sm line-height-sans-4 margin-bottom-4 tablet:margin-bottom-0"
+              }
+            >
+              {t("pageHeaderParagraph")}
+            </p>
+          </Grid>
+          <Grid
+            className="display-none tablet:display-flex tablet:flex-justify-center tablet:flex-align-center"
+            tablet={{ col: 6 }}
+          >
+            <div className="display-flex flex-justify-center flex-align-center">
+              <img
```

Comment:
Is there value in using the next js `<Image />` here? https://nextjs.org/docs/pages/api-reference/components/image

Reply:
Yes, esp since it sets the `srcset`. We haven't used content images before, so not sure what the right initial size and sizes should be. Will take a look.